enhancements/declarative-index-config.md

```diff
@@ -69,11 +69,12 @@ A human consumable representation of packages in an index in the form of json/ya
 As a user with pull permission from the namespace an index is hosted in/as a component of OLM, I can query an index for a json/yaml representation of the packages of an index.
 The representation of individual packages will allow individual package owners to reason about/make changes to their individual packages in isolation.  
 
-`opm index inspect --image=docker.io/my-namespace/community-operators --output=json`
+`opm unpack docker.io/my-namespace/community-operators`
 
 pulls the index image from the registry, creates a new folder `community-operators` and copies the package representations for the packages that are in the image to the local folder. 
 
 ```bash
+$ opm unpack docker.io/my-namespace/community-operators
```

Comment:
does this command work even if the catalog image in question only contains an index.db?

Reply:
I think the intention is for this command to work with config only. Anik can share his thought here. We have an equivalent command for db-based index which is `opm index extract` or something.

Reply:
It'll be fairly simple to unpack an index whether it contains index.db or configs. I don't see why we might be opposed adding extraction of index.db by `unpack` (probably doesn't fit the narrative of this enhancement though so not sure if we want to call this out here, or just go ahead and bake the index.db extraction in)

Reply:
the context of my question is "what does the process look like to migrate an existing catalog/index.db to declarative config format".  So i don't necessarily care whether it's `opm unpack`(transparent logic) `opm unpack --index-format` or `opm migrate-index-db` or something else, just trying to figure out what the story will be.

Reply:
In that case, the migration plan remains the same as stated here https://github.com/operator-framework/enhancements/blob/master/enhancements/declarative-index-config.md#migration-plan
at least in the context of this PR to update the enhancement. 

To reiterate: 
Once we deprecate all of the existing commands, and provide the `opm migrate` command, we will be auto migrating any remaining indexes that have index.db in them to configs (following which `opm unpack` can be used to extract the configs dir for the index). 
PS: when the indexes are auto migrated, there will likely be configs based implementation of the existing deprecated commands (before they are removed in the following release)